hbase-common/src/main/java/org/apache/hadoop/hbase/HBaseConfiguration.java

```diff
@@ -85,16 +85,69 @@ public static Configuration addHbaseResources(Configuration conf) {
     return conf;
   }
 
+  public static Configuration create() {
+    return create(true);
+  }
+
   /**
    * Creates a Configuration with HBase resources
    * @return a Configuration with HBase resources
    */
-  public static Configuration create() {
+  public static Configuration create(boolean lowLimits) {
     Configuration conf = new Configuration();
     // In case HBaseConfiguration is loaded from a different classloader than
     // Configuration, conf needs to be set with appropriate class loader to resolve
     // HBase resources.
     conf.setClassLoader(HBaseConfiguration.class.getClassLoader());
+
+
+    if (lowLimits) {
+
+      // can't set this for every test currently
+      // conf.set("hbase.regionserver.hostname", "127.0.0.1");
+
+      conf.setInt("hbase.hfilearchiver.thread.pool.max", 2);
+      conf.setInt("hbase.loadincremental.threads.max", 3);
```

Comment:
This one is sized by processor count. Rather than hard-code it, keep it based on processor count but divide by 2 or 4 w/ minimum 3?